doc/guides/compile.md

```diff
@@ -338,6 +338,47 @@ When debugging mode is enabled
   * Because `-g` flag would be added to `mrbc` runner.
     * You can have better backtrace of mruby scripts with this.
 
+### Additional license terms
+
+The license terms set in the `conf.terms` property are printed as part of
+the `<build-dir>/LICENSE.yml` file.
+This initial value is an empty array (`[]`). If you want to add it, do the
+following:
+
+```ruby
+conf.terms << "path/to/ADDITIONAL-LICENSE"
+```
+
+The file path can be specified as an absolute path as well as a relative
+path based on the build directory.
+Also, you can give not only the file path but also a hash that combines
+the name and the content of the terms.
+
+Objects that can be specified directly in `conf.terms` are `nil`, strings
```

Comment:
```suggestion
Optional parameters that can be specified to `conf.terms`:

- `nil`
- strings that represent paths
- array or hash
- any object that responds to `#to_path` and `#read` (assuming file and pathname)

```

Reply:
The `terms` property is now just a string, so it's no longer explained.